concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -3424,7 +3443,17 @@ jobs:
                   fi
 
                   cf cancel-deployment cloudwatch-exporter || true
-                  cf push --strategy=rolling cloudwatch-exporter
+                  cf push cloudwatch-exporter-staging -f staging-manifest.yml --no-start  --no-route
+                  cf stage cloudwatch-exporter-staging
+
+                  DROPLET=$(cf droplets cloudwatch-exporter-staging | tail -n +4 | awk '{print $1}' | head -n 1)
+
+                  cf download-droplet cloudwatch-exporter-staging --droplet "${DROPLET}" --path droplet.tgz
+
+                  set -o pipefail
+                  # mask the aws secret in the concourse output
+                  cf push cloudwatch-exporter -f manifest.yml --droplet droplet.tgz --strategy rolling 2>&1 | \
+                    sed "s/${YACE_AWS_SECRET_ACCESS_KEY}/********/g"
```

Comment:
I'm not sure an AWS key is safe to put in a sed pattern, all sorts of funny characters in there.

Reply:
The stackoverflow rabbit-hole for how to do this robustly without installing more packages is depressing.

The simplest thing might be to just use `grep -Fv` and accept that matching lines will just be silently swallowed.

Reply:
Okay updated to a:
```
sed -r "s/AWS_SECRET_ACCESS_KEY: .+/AWS_SECRET_ACCESS_KEY: ********/g"
```